swap_meet/vendor.py

```diff
@@ -1,2 +1,112 @@
+from swap_meet.item import Item
```

Comment:
This import is actually not needed! It's a good idea to check to make sure VS Code didn't automatically add imports you don't need.